### ISSUE TITLE
background music

### ISSUE THREAD

**Issue by silverlona** [biner88/metronome#14]
Hi, I use the metronome to mark the pace in a training app. However, I want background music and I can't get the metronome to work with for example spotify playing music at the same time. Is this possible or something that can be fixed in the next version?

I can off course write my own code for this but the metronome works great except for this issue described above! I don't see that it would be a negative thing to enable this feature. Thanks for a great package!

best regards

silverlona

**Comment by silverlona** [comment]
exactly, Spotify stop when I start the app. if I comment out all the metronome code it works fine. I also use the flutter_tts package in this app and flutter_tts works fine with my app and Spotify.

**Comment by silverlona** [comment]
by the way, I only use IOS at this stage so I have not tried with Android yet!

**Comment by biner88** [comment]
Add enableSession: false in init and try.

**Comment by silverlona** [comment]
ok, great, thanks! it works! I did see enableSession in the definition but I did not connect it to my issue! once again thanks for you super fast response! all the best /silverlona
